### DISCUSSION TITLE
Upgrading from 2.5.1 to 2.52 breaks livewire response serialisation

### DISCUSSION THREAD

**Discussion by slakbal** [livewire/livewire#3270] [7. Bugs (archived)] [answered]
Hi,

For actions on a table, I have the following blade code:

```blade
<a wire:click="$emit('openModal', 'tenant.deactivate-modal', {{ json_encode(['tenant' => $tenant->id]) }})" wire:loading.attr="disabled" role="button" href="#" title="{{ __('Deactivate') }}">
    Deactivate
</a>
```

The above generates the following:

```html
<a wire:click="$emit('openModal', 'tenant.deactivate-modal', {"tenant":1})" wire:loading.attr="disabled" role="button" href="#" title="Deactivate">
    Deactivate
</a>
```

For the tables I'm using: https://github.com/rappasoft/laravel-livewire-tables
For the modals I'm using: https://github.com/livewire-ui/modal

Everything works with Livewire v2.5.1, the modal is called up and the action performed and livewire responds with the updated part. But once I upgrade to v2.5.2 the initial reder is correct, the modal is called up but after the component action is finished livewire responds with the following, which is clearly a broken response and it feels like an escaping/serialisation issue:

```html
<a wire:click="$emit('openModal', 'tenant.deactivate-modal', {" wire:loading.attr="disabled" role="button" href="#" title="Deactivate">
    Deactivate
</a>
```
Not 100% sure why this is happening, any ideas? This only happens when I move from v2.5.1 to v2.5.2. 

Feels like a bug, but maybe I missed something. Any assistance would be appreciated!

Thanks!

**Comment by joshhanley** [comment, marked as answer]
@slakbal there was a bug in 2.5.2 causing another issue, which has been fixed.

So can you please upgrade to 2.5.3 and confirm if your issue is still happening or not?

Thanks!

**Reply by slakbal** [reply]
Much appreciated!
